### PR TITLE
Docs: Align the content in the "overview" page of the "pre-installation requirements" chapter  with the chapter's structure

### LINKED ISSUE #25: [Bug] misleading content in Installation Guide document
**Describe the bug**
In the Service Workbench Installation Guide document, the overview page of the pre-installation requirements chapter describes the document structure for this chapter. However, the document structure described in the overview is inconsistent with the actual document structure, which is very misleading and makes deployment unnecessarily difficult for new users.
Specifically, clicking "pre-installation requirements" in the original overview page will jump to the "tool requirements" page, and this page does only contain the content related to tool requirements, not as described in the overview (Also contains "software requirements" related content). The relevant content of the latter is given in a separate page, and this page has no corresponding link in the overview.

**To Reproduce**
1. Visit [the page mentioned below](https://awslabs.github.io/service-workbench-on-aws-cn/installation_guide/installation/pre-installation/overview)
2. See an overview **NOT** consistent with the document structure of this chapter

**Expected behavior**
See an overview consistent with the document structure of this chapter

**Screenshots**
Actual: 
image
Expected:
image

**Versions (please complete the following information):**
 - v5.1.0

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by ghost**
Issue #25

Description of changes:

Modified the overview page of the pre-installation requirements chapter to align its content with the structure of the chapter

Checklist:

- [x] Have you successfully tested with your changes locally?
  - for preview, see [the page of my repo](https://6zkf6zs05ryz.github.io/service-workbench-on-aws-cn/installation_guide/installation/pre-installation/overview)

### CODE REVIEW

File: docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-id: instance-req
-title: Instance requirements
-sidebar_label: Instance requirements
+id: ec2-instance-req
+title: EC2 Instance requirements
```
Comment (zxkane) [inline docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md]: @dengmingtong do you think it's necessary?

**Review by zxkane — CHANGES_REQUESTED** [review]

File: docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md (hunk 1 above)
Comment (dengmingtong) [reply docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md]: I think it's better to keep title and sidebar.

Comment (ghost) [reply docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md]: The modification here is mainly to make the title in the table consistent with the title in the sidebar.

image

Comment (zxkane) [reply docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md]: It's something I prefer to keeping sync with upstream.

Could you submit a PR to fix the table style in upstream. Also create a bug for section title and sidebar mismatch. Let's see the feedback from upstream.

Comment (ghost) [reply docs/docs/installation_guide/installation/pre-installation/ec2-instance-req.md]: I created this [PR](https://github.com/awslabs/service-workbench-on-aws/pull/1006) and will update here as soon as there are any comments.

**Comment by ghost** [comment]
Upstream has not responded. Considering this matter is not in a hurry, I will close the PR on both sides first.

**Comment by zxkane** [comment]
Let's using this one fixing table format only.

**Comment by ghost** [comment]
fixed and only fixed table format

**Review by zxkane — COMMENTED** [review]
I do not see format issue of table in latest built version. Is it already fixed?

**Review by dengmingtong — APPROVED** [review]

**Comment by ghost** [comment]
> I do not see format issue of table in latest built version. Is it already fixed?

The following issues described in issue #25 still exist in the latest commit:
1. The software req in the sidebar is an independent section, but it is mistakenly divided into a part of the pre installation req in the table;
    image
2. The table claims that the pre installation section contains content about aws tool+software+access, etc., but in fact, the pre installation req in the table is bound to the tool req in the sidebar, which only contains aws tool related content;
3. The title of the instance section in the table does not match the actual ID (either with the EC2 prefix or none).

@zxkane

**Review by zxkane — APPROVED** [review]

**Pull request merged**